### PR TITLE
Steam integration toggle & Stepmania/Etterna mappack import

### PR DESCRIPTION
**Pull request by tasgon**
Basically what the title says. This lets people disable integration with Steam via the `DISABLE_STEAM` environment variable. It also adds the ability to import mappacks in the form of zip files.

### CODE REVIEW

**Comment by Illuminati-CRAZ** [comment]
It appears there's also something related to showing a grade for a map using local score if the map isn't submitted

**Comment by tasgon** [comment]
Oh, yes, I felt like that might be desirable, as people playing without Steam integration would probably still want to see their local scores in the song listing. (Or at least I would, what do you think?)

**Comment by Illuminati-CRAZ** [comment]
I believe Swan would prefer if there was one pull request per feature added.
(And other things according to [the submission guidelines](https://github.com/Quaver/Quaver/blob/ui-redesign/.github/CONTRIBUTING.md#submitting-a-pull-request) and code style and whatnot)

I agree that using the local score in song listing when global isn't available is a good idea.

File: Quaver.Shared/Screens/Settings/SettingsDialog.cs [hunk 1]
```diff
@@ -456,6 +456,7 @@ private void CreateSections()
                     new SettingsBool(this, "Display Failed Local Scores", ConfigManager.DisplayFailedLocalScores),
                     new SettingsBool(this, "Display Online Friends Notification", ConfigManager.DisplayFriendOnlineNotifications),
                     new SettingsBool(this, "Display Song Request Notifications", ConfigManager.DisplaySongRequestNotifications)
+                    //new SettingsSlider(this, "Number of threads to use for mapset importing", ConfigManager.)
```
Comment (IceDynamix) [inline Quaver.Shared/Screens/Settings/SettingsDialog.cs:459]: Probably remove this comment

**Comment by Swan** [comment]
Please split this into multiple pull requests/commits.

**Pull request closed without merging**